ha_client.py

```diff
@@ -196,6 +196,34 @@ def find_entity_attr(self, entity):
                     return entity_attr
         return None
 
+    def list_entities(self, types):
+        """List all entities matching domains used within our skill
+        and return them as list.
+
```

Comment:
Would be great to add `Args` and `Returns` sections to this docstring so we know exactly what inputs and outputs to expect.

For the `Exceptions` this should go under a `Raises` section.
You can see some examples here: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Perhaps use static typing an mypy format? `def list_entities(self, types: List[str]) -> Dict[str, str]:` or combine?

Reply:
yeah if you have the types specified in the signature then they don't need to be duplicated in the Args and Returns section, eg:
```python
def function_with_pep484_type_annotations(param1: int, param2: str) -> bool:
    """Example function with PEP 484 type annotations.

    Args:
        param1: The first parameter.
        param2: The second parameter.

    Returns:
        The return value. True for success, False otherwise.

    """
```

This is the standard we've been using across Mycroft repo's so would be good to keep it consistent:
https://github.com/MycroftAI/.github/blob/main/CONTRIBUTING.md#documentation